src/components/SongList.js

```diff
@@ -90,7 +90,7 @@ class SongList extends Component {
             ]
             return (
               <ContextMenuTrigger id={ song.hash || song.hashMd5 }>
-                <SongListItem key={ makeRenderKey(songTags) } title={ song.songName } ratings={ song.ratings } artist={ song.authorName } uploader={ song.uploader } difficulties={ song.difficulties || song.difficultyLevels } imageSource={ song.coverUrl } songKey={ song.key } hash={ song.hash || song.hashMd5 } file={ song.file } downloads={ song.downloadCount } upvotes={ song.upVotes } downvotes={ song.downVotes } plays={ song.playedCount } />
+                <SongListItem key={ makeRenderKey(songTags) } title={ song.songName } ratings={ song.ratings } artist={ song.authorName } uploader={ song.uploader } difficulties={ song.difficulties || song.difficultyLevels } imageSource={ song.coverUrl } songKey={ song.key } hash={ song.hash || song.hashMd5 } file={ song.file } downloads={ song.downloadCount } upvotes={ song.upVotes } downvotes={ song.downVotes } plays={ song.playedCount } uploadDate={ !!song.createdAt && !!song.createdAt.date && !!song.createdAt.timezone ? new Date(Date.parse(song.createdAt.date + " " + song.createdAt.timezone)).toLocaleString() : '' } />
```

Comment:
Probably not the best place to parse the date, but I didn't really know where else to do it